docs/smoke-test.md

```diff
@@ -0,0 +1,14 @@
+## How to run smoke-test:
+
+1. Switch to _scripts/smoke-test/_ folder
+1. Connect some device
+1. Run command
+```
+python teaser_smoke_test.py
+```
+
+#### You can add such params:
+
+_b_ - Build new Teaser version. Can be **true** or **false**
```

Comment:
What if this flag is false?

Reply:
We install exist app or crash if it not exists. 
Maybe I should change default value to _true_